### PR TITLE
#100 build upload docker

### PR DESCRIPTION
**Pull request by jiuliangut**
- updated URI for the data_upload image to the lambda terraform 
- Updated pdf file naming convention in `data_upload/extract.py` - pdfs will now be named after the date 7 days prior to the run date, so that a given file is named after the first day of data stored in it

### CODE REVIEW

File: terraform/data_upload_lambda.tf [hunk 1]
```diff
@@ -52,8 +52,8 @@ resource "aws_lambda_function" "c14-earthquake-report-lambda" {
   function_name    = "c14-earthquake-report-lambda"
   role             = aws_iam_role.c14_lambda_execution_role.arn
   package_type     = "Image"
-  image_uri        = "${var.ETL_ECR_URI}:latest" #placeholder until we get the actual image uri
-  timeout          = 900 # 15 minutes
```
Comment (S1mpySloth) [inline terraform/data_upload_lambda.tf:56]: Correctly updated image URI

**Review by S1mpySloth — APPROVED** [review]
Everything looks good, terraform updated with correct Image URI and pdf naming convention also updated.

**Review by Jakub-Poskrop — DISMISSED** [review]
Looks good

**Review by Jakub-Poskrop — APPROVED** [review]
Still looks good.

**Review by Fahi28 — APPROVED** [review]
nice terraforming

**Review by S1mpySloth — APPROVED** [review]
Small tweak to resolve conflict, looks good!

**Pull request merged**